Bitcoin Core and CLI Tutorial

rishkwal opened the pull request:
Closes #331

mojoX911 reviewed (commented):
Ack..

Can we add two more section. The data directory structure, and the config file.

At datadir, just give abrief tour of what each file does. And tell them to always backup the wallet files.

At config explain in one lines what each config does. 

rest is good to go.

rishkwal commented:
> Ack..
> 
> Can we add two more section. The data directory structure, and the config file.
> 
> At datadir, just give abrief tour of what each file does. And tell them to always backup the wallet files.
> 
> At config explain in one lines what each config does.
> 
> rest is good to go.

Updated the command, and added a section for Data Directory, Config and Wallets

mojoX911 reviewed (approved):
Ack
